packaging/aznfs/DEBIAN/postinst

```diff
@@ -35,6 +35,22 @@ chattr +i /opt/microsoft/aznfs/randbytes
 set -e
 
 # Start aznfswatchdog service.
-systemctl daemon-reload
-systemctl enable aznfswatchdog
-systemctl start aznfswatchdog
\ No newline at end of file
+init="$(ps -q 1 -o comm=)"
+
+if [ "$init" == "systemd" ]; then
+    echo "System is booted with systemd, continuing installation!"
+    
+    # Reload systemd configuration to ensure any changes take effect.
+    systemctl daemon-reload
+    
+    # Enable the 'aznfswatchdog' service to start on boot.
+    systemctl enable aznfswatchdog
+    
+    # Start the 'aznfswatchdog' service.
+    systemctl start aznfswatchdog
+else
+    echo "OK, systemd installed, but not booted, continuing installation!"
```

Comment:
This same log line will come twice in the same panel if kept here as well since it is already there in the end of preinst script. Remove it.